docs/UsersGuide/source/RunSRW.rst

```diff
@@ -988,14 +980,14 @@ If the login shell is csh/tcsh, it can be set using:
 Automated Option
 ^^^^^^^^^^^^^^^^^^^
 
-The simplest way to run the Rocoto workflow is to automate the process using a job scheduler such as :term:`Cron`. For automatic resubmission of the workflow at regular intervals (e.g., every minute), the user can add the following commands to their ``config.yaml`` file *before* generating the experiment:
+The simplest way to run the Rocoto workflow is to automate the process using a job scheduler such as :term:`Cron`. For automatic resubmission of the workflow at regular intervals (e.g., every 2 minutes), the user can add the following commands to their ``config.yaml`` file *before* generating the experiment:
 
 .. code-block:: console
 
    USE_CRON_TO_RELAUNCH: true
-   CRON_RELAUNCH_INTVL_MNTS: 3
+   CRON_RELAUNCH_INTVL_MNTS: 2
```

Comment:
Given @JeffBeck-NOAA's feedback in ContainerQuickstart.rst, should this be set to a value of 3 (or higher)?